report/index.html

```diff
@@ -0,0 +1,5176 @@
+<!DOCTYPE HTML>
```

Comment:
<img width="2048" alt="image" src="https://user-images.githubusercontent.com/36746022/208458575-7e637cb3-b5ce-4520-bbe7-a6f3806f3c1d.png">
this report is up to date ?

Should  rename report title from `Behave Test Report` to `Behave Test NodoDei Pagamenti Report <ExecutionDate>`

who upload this report on `pagopa`<d|u>`nodotestsa` storage ?

Reply:
the reports you see are only for local debugging (the html ones, so they are all old). The official report is the one on azure pipeline as shown during demos.

Reply:
ok the remove this useless file .
you can get confused